### ISSUE TITLE
Are you open for contributors?

### ISSUE THREAD

**Issue by herou** [SwedbankPay/swedbank-pay-sdk-android#137]
Hello,

The subject describes itself. Are you open to contributors?

**Comment by asbjornu** [comment]
Yes, absolutely. What sort of contributions were you thinking of making?

**Comment by herou** [comment]
@asbjornu  I an android developer that's why I found this repo written in Kotlin.Coding would be nice, something small to start.
